sample/lib/ui/pages/main_page.dart

```diff
@@ -40,7 +40,9 @@ class _MainPageState extends State<MainPage> {
 
     return BlocListener<PageBloc, PageState>(
       listener: (BuildContext context, PageState state) {
+        _selectedPage = state;
```

Comment:
@onurccn state.currentPage olmasi gerekiyor sanirim?

Reply:
normalde oyle olmasi lazimdi ama selectedPage in tipini PageState olarak degistirmissiniz sanirim. o yuzden tum state i yeniledim listenerda. Ayrica bisey farkettim. PageState degistiginde build fonksiyonuna sifirdan girmiyo icerdeki state i degistiriyo sadece. Oncesinde de stateful widget olmasina ragmen sifirdan build ediyordu sayfayi.